components/Pagination/utils/pagination.js

```diff
@@ -71,7 +71,7 @@ const pagination = ({
   if (penultimatePage < lastPage - 2) {
     visiblePages = [...visiblePages.slice(0, -1), '...'];
 
-    if (showLastPagination) {
+    if (!hideLastPagination) {
       visiblePages = [...visiblePages, lastPage];
```

Comment:
```suggestion
    if (hideLastPagination) {
      visiblePages = [...visiblePages, lastPage];
```

you should adjust the logic with a hideLastPagination being true.